includes/emails/class-emails.php

```diff
@@ -587,5 +588,30 @@ public static function get_password_reset_url( $user, $key ) {
 
 		return wp_lostpassword_url();
 	}
+
+	/**
+	 * Trigger an update to all email template posts when theme color is updated in customizer.
```

Comment:
The colors can also be changed in the Site Design wizard:

<img width="1600" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/7383192/08c037cf-d8c3-40a6-9f0c-d34afc0ba773">

Theme mods are stored in options, so an option value update could be a better event to listen for.

Reply:
Good catch! I've updated to use the `update_option_theme_mods_{theme}` hook in https://github.com/Automattic/newspack-plugin/pull/3052/commits/bb074670a08299814fc0cc365fdc46195de9276d